src/data/emacs-mode/agda2-mode.el

```diff
@@ -1306,7 +1306,8 @@ top-level scope."
 
 (agda2-maybe-normalised
  agda2-elaborate-give
- "Elaborate check the given expression against the hole's type and fill in hole with the elaborated term"
+ "Elaborate check the given expression against the hole's type and fill in the
+ hole with the elaborated term"
```

Comment:
What is the reason for this change?  Is this connected to the nix-build somehow?

Reply:
Because of the nixpkgs update, we are now affected by https://github.com/emacs-mirror/emacs/commit/0ebea8ffbfb7b9b1bd92f30011df0875b54eb663

Reply:
Thanks for the clarification!